spec/bolt/transport/local_spec.rb

```diff
@@ -131,7 +131,7 @@
     end
 
     context 'with slow input' do
-      let(:file_size) { 100 }
+      let(:file_size) { 10 }
```

Comment:
This is an unrelated change but it cuts this test from 20 seconds to 2 seconds and I don't _think_ it has an impact on the behavior of the test. I couldn't quite tell what problem it was meant to spot though, so I can't be sure.

Reply:
This seems totally reasonable.